### PR TITLE
fix app keeps restarting & unable to run `CrashActivity`

### LINKED ISSUE #1332: Crash screen won't show up after PR #1237
### Steps to reproduce

1. Try create some crash situation, maybe throw some exceptions.

### Expected behavior

- `CrashScreen` should show
- Crash event should still be pushed to Firebase
- App should continue to work.

### Actual behavior

2. App keeps restarting (can see in log also), showing blank screen with/without dialog message something like "App keeps close, Wait/Close".
3. `CrashScreen` won't show.
4. Must manually close the app to be able to use it again.

### Crash logs

[API-34-Android-14_2024-10-17_085604.logcat.zip](https://github.com/user-attachments/files/17404963/API-34-Android-14_2024-10-17_085604.logcat.zip)

### Mihon version

latest dev build

### Android version

Android 14

### Device

AVD

### Other details

This happens after PR #1237 adding option to opt-out of Firebase crashlytics/analytics.

But before that PR, I think the `GlobalExceptionHandler` trying to intercept exception in order to show `CrashActivity` and eventually didn't actually report any event to Firebase.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by cuong-tran**
close #1332

### CODE REVIEW

File: app/src/standard/java/mihon/core/firebase/Firebase.kt [hunk 1]
```diff
@@ -14,7 +14,7 @@ object Firebase {
             FirebaseAnalytics.getInstance(context).setAnalyticsCollectionEnabled(enabled)
         }.launchIn(scope)
         preference.crashlytics().changes().onEach { enabled ->
-            FirebaseCrashlytics.getInstance().setCrashlyticsCollectionEnabled(enabled)
+            FirebaseCrashlytics.getInstance().isCrashlyticsCollectionEnabled = enabled
```
Comment (AntsyLich) [inline app/src/standard/java/mihon/core/firebase/Firebase.kt:17]: It's the same thing

Comment (cuong-tran) [reply app/src/standard/java/mihon/core/firebase/Firebase.kt:17]: yes, same, just that ktlint or Android Studio suggests using that form. Should I revert it?

Comment (AntsyLich) [reply app/src/standard/java/mihon/core/firebase/Firebase.kt:17]: Yes please

**Comment by AntsyLich** [comment]
Fixed the root cause

**Pull request closed without merging**